### PR TITLE
added celebrities controller

### PR DESCRIPTION
**Pull request by msachinidis**
made celebrity controller

### CODE REVIEW

File: app/controllers/celebrities_controller.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  def new
+    @celebrity = Celebrity.new
+  end
+
+   def edit
+    @celebrity = Celebrity.find(params[:id])
+  end
+
+  def update
+  end
```
Comment (yasirrafiq786) [inline app/controllers/celebrities_controller.rb:20]: You have to add lines here

File: app/controllers/celebrities_controller.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+ def create
```
Comment (yasirrafiq786) [inline app/controllers/celebrities_controller.rb:22]: We need to find a way to link User to celebrtity

Comment (azleong) [reply app/controllers/celebrities_controller.rb:22]: @celebrity.user = @user ?

Comment (yasirrafiq786) [reply app/controllers/celebrities_controller.rb:22]: Yes thats true. But the idea is that when celebrity is being created by some ##user which is already ##signed-in so we would be able to just find it through params.

**Pull request merged**